README.md

```diff
@@ -1,2 +1,290 @@
-# Project-Hello
-A project to master tools and best practices for future projects
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    <!-- - [Tech Stack](#tech-stack) -->
+    <!-- - [Key Features](#key-features) -->
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+    <!-- - [Usage](#usage) -->
+    <!-- - [Run tests](#run-tests) -->
+    <!-- - [Deployment](#triangular_flag_on_post-deployment) -->
+- [👥 Authors](#authors)
+<!-- - [🔭 Future Features](#future-features) -->
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  <!-- - [🙏 Acknowledgements](#acknowledgements) -->
+  <!-- - [❓ FAQ](#faq) -->
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello_Project] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
+**[Hello_Project]** is a... project to master tools and best practices for future projects
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+
+<!-- ### Tech Stack <a name="tech-stack"></a> -->
+
+<!-- > Describe the tech stack and include only the relevant sections that apply to your project. -->
+
+<!-- <details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details> -->
+
+<!-- <details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details> -->
+
+<!-- <details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details> -->
+
+<!-- Features -->
+<!--
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
+
+- **[key_feature_1]**
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p> -->
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Add a link to your deployed project.
+
+- [Live Demo Link]
```

Comment:
- [ ] [REQUIRED] Logically I suggest you put `"Coming soon"` text in this section because Your project's deploy link does not exist 😃